scripts/variantstore/wdl/GvsVQSRClassic.wdl

```diff
@@ -0,0 +1,203 @@
+version 1.0
+
+import "GvsWarpTasks.wdl" as Tasks
+import "GvsUtils.wdl" as Utils
+
+workflow JointVcfFiltering {
+  input {
+    String base_name
+    Int num_samples_loaded
+    File sites_only_variant_filtered_vcf
+    File sites_only_variant_filtered_vcf_idx
+    Array[File] sites_only_variant_filtered_vcfs
+    Array[File] sites_only_variant_filtered_vcf_idxs
+
+    Array[String] indel_recalibration_annotation_values = ["AS_FS", "AS_ReadPosRankSum", "AS_MQRankSum", "AS_QD", "AS_SOR"]
+    Array[String] snp_recalibration_annotation_values = ["AS_QD", "AS_MQRankSum", "AS_ReadPosRankSum", "AS_FS", "AS_MQ", "AS_SOR"]
```

Comment:
does the order matter? It feels odd to me that they are two almost identical lists, but the seem very different at first glance

Reply:
yeah, I have the same comment on `GvsCreateFilterSet.wdl` above.

Reply:
Yup. Commented appropriately